src/bomsquad/vulndb/client/nvd.py

```diff
@@ -31,10 +34,10 @@ def vulnerabilities(
         while True:
             url = f"{self.CVE_STEM}?startIndex={offset}"
             if last_mod_start_date:
-                url += f"&lastModStartDate={last_mod_start_date.isoformat()}&lastModEndDate={datetime.now().isoformat()}"
-                logger.info(
-                    f"Querying from {offset} - {limit} and {last_mod_start_date.isoformat()} - {datetime.now().isoformat()}"
-                )
+                dtstart = (last_mod_start_date + timedelta(microseconds=1000)).isoformat()
+                dtend = datetime.utcnow().replace(tzinfo=timezone.utc).isoformat()
```

Comment:
Using utcnow() instead of now(), adding 1000 microseconds to skip querying already fetched CVEs.

Reply:
For timezone-aware datetime in utc, it would be a little more direct and clear to datetime.now(timezone.utc) rather than utcnow with replace.